dds_cli/file_handler.py

```diff
@@ -56,12 +56,19 @@ def __init__(self, user_input, local_destination, project=None):
 
     # Static methods ############ Static methods #
     @staticmethod
-    def append_errors_to_file(file: pathlib.Path, info):
+    def append_errors_to_file(log_file: pathlib.Path, file, info, status):
         """Save errors to specific json file."""
+
+        failed_to_save = {
+            str(file): {
+                **FileHandler.make_json_serializable(non_json=info),
+                "status": FileHandler.make_json_serializable(non_json=status),
+            }
+        }
         try:
-            with file.open(mode="a") as errfile:
+            with log_file.open(mode="a") as errfile:
                 json_output = json.dumps(
-                    info,
+                    failed_to_save,
                     indent=None,
```

Comment:
Maybe provide an indent value so the file is a bit easier to read?

Reply:
Done!